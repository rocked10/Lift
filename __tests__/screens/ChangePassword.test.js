import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import ChangePassword from '../../screens/changePassword';

describe('Testing Change Password Screen', () => {
    const wrapper = renderer.create(<ChangePassword />);

    it('Should render', () => {
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});