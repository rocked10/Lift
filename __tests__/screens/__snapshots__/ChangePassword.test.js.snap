// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Change Password Screen Should render 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#D3D3D3",
        "borderRadius": 25,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 20,
        "padding": 20,
        "width": "90%",
      }
    }
  >
    <Text />
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Enter a new password..."
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "flex": 1,
          "height": 50,
          "marginLeft": 10,
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#D3D3D3",
        "borderRadius": 25,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 20,
        "padding": 20,
        "width": "90%",
      }
    }
  >
    <Text />
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Confirm your new password..."
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "flex": 1,
          "height": 50,
          "marginLeft": 10,
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
        "borderColor": "transparent",
        "borderRadius": 4,
        "borderStyle": "solid",
        "borderWidth": 0,
        "elevation": 0,
        "minWidth": 64,
      }
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "disabled": undefined,
        }
      }
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Object {
            "borderRadius": 4,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
            },
            undefined,
          ]
        }
      >
        <Text
          numberOfLines={1}
          selectable={false}
          style={
            Array [
              Object {
                "color": "#000000",
                "fontFamily": "System",
                "fontWeight": "400",
              },
              Object {
                "textAlign": "left",
              },
              Array [
                Object {
                  "letterSpacing": 1,
                  "marginHorizontal": 16,
                  "marginVertical": 9,
                  "textAlign": "center",
                },
                undefined,
                Object {
                  "textTransform": "uppercase",
                },
                Object {
                  "color": "#6200ee",
                  "fontFamily": "System",
                  "fontWeight": "500",
                },
                Object {
                  "fontFamily": "System",
                  "fontWeight": "500",
                },
                undefined,
              ],
            ]
          }
        >
          Change Password
        </Text>
      </View>
    </View>
  </View>
</View>
`;
